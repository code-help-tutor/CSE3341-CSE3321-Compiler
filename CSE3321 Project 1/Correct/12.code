pRocedure In oUt iF TheN END BEGIN EnD ref EOS