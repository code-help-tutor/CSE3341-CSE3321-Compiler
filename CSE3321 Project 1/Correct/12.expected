ID[pRocedure]
ID[In]
ID[oUt]
ID[iF]
ID[TheN]
ID[END]
ID[BEGIN]
ID[EnD]
ID[ref]
ID[EOS]
